### ISSUE TITLE
Line breaks in attributes causing problems in pagination

### ISSUE THREAD

**Issue by tomkiss** [putyourlightson/craft-sprig#264]
Hey there.

I'm using Craft 4 and having a play with Sprig for the first time and soon ran into the following issue.

Given the code from the [pagination example](https://putyourlightson.com/sprig/cookbook?page=5#pagination):

```twig
<button sprig s-val:page="{{ page + 1 }}" s-push-url="?page={{ page + 1 }}">
      Next
</button>
```

If the following markup is used:

```twig
<button class="someclass
anotherclass" sprig s-val:page="{{ page + 1 }}" s-push-url="?page={{ page + 1 }}">
      Next
</button>
```

...the pagination example will fail. There is no error, but the markup will not work.

Although it's not best practice to use line breaks in HTML attributes, it is not invalid HTML, and I would not have expected it to fail silently.

Anyway, I hope this is helpful in some way.

**Comment by bencroker** [comment]
Thanks for reporting this, I'll look into releasing a fix.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-sprig-core/commit/9dfc817fb7637915b761187f8702fc271e91d0f1 and released in Sprig Core 2.0.6, `composer update` should update it for you. Thanks again!
